sparse_list.py

```diff
@@ -125,22 +135,22 @@ def __mul__(self, multiplier):
 
     def count(self, value):
         '''
-        return number of occurrences of value
+        Return the number of occurrences of value.
```

Comment:
Urgh. I hesitated for a while before changing the text, but then my inner grammarian assumed control, i guess. I can revert it back, I don't know how to choose between consistency with the existing doc and literacy. _You_ choose. :)